docs/scripting/custom-command.md

```diff
@@ -70,15 +70,15 @@ On our manifest, we added script module. The `entry` is where our script is stor
 
 ## Creating Custom Commands
 
-Now this is the fun part, creating our custom commands. First, we gonna add the module
+Now comes the fun part- creating our custom commands. First, we will add the module.
```

Comment:
Should have a space before the `-`, makes it cleaner

`Now comes the fun part- creating our custom commands`

`Now comes the fun part - creating our custom commands`

Reply:
Updated the PR to fix this issue.